### PR TITLE
updated boilerplate files and Makefile

### PR DESCRIPTION
**Pull request by Syd-Vigilant**
updated boilerplate files and Makefile

### CODE REVIEW

**Comment by Syd-Vigilant** [comment]
Please test this

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
retest this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
retest this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by maxwelllincoln** [comment]
test this please

**Comment by chrislovecnm** [comment]
test this please

**Comment by geojaz** [comment]
Please test this

**Comment by geojaz** [comment]
test this please

File: test/make.sh [hunk 1]
```diff
@@ -18,8 +18,7 @@
 # shebang has a '- e' flag, which causes it
 # to exit on error
 function check_bash() {
-find . -name "*.sh" -not -path "*/istio-1.0.1/*" \
-  -not -path "*/gke-istio-shared/*" | while IFS= read -d '' -r file;
+find . -name "*.sh" | while IFS= read -d '' -r file;
```
Comment (geojaz) [inline test/make.sh]: We need to ignore the gke-istio-shared files. These should be checked in that repo. once they get here, we should think of them as a black box. It's just a third party library

File: test/make.sh [hunk 2]
```diff
@@ -42,52 +41,45 @@ function basefiles() {
 # every file named 'Dockerfile'
 function docker() {
   echo "Running hadolint on Dockerfiles"
-  find . -name "Dockerfile" -not -path "*/istio-1.0.1/*" \
-    -not -path "*/gke-istio-shared/*" -exec hadolint {} \;
+  find . -name "Dockerfile" -exec hadolint {} \;
```
Comment (geojaz) [inline test/make.sh]: as above

File: test/make.sh [hunk 3]
```diff
@@ -42,52 +41,45 @@ function basefiles() {
...
 }
 
 # This function runs 'terraform validate' against all
 # files ending in '.tf'
 function check_terraform() {
   echo "Running terraform validate"
   #shellcheck disable=SC2156
-  find . -name "*.tf" -not -path "*/istio-1.0.1/*" \
-    -not -path "*/gke-istio-shared/*" -exec bash -c 'terraform validate --check-variables=false $(dirname "{}")' \;
+  find . -name "*.tf" -exec bash -c 'terraform validate --check-variables=false $(dirname "{}")' \;
```
Comment (geojaz) [inline test/make.sh]: see above

File: test/make.sh [hunk 4]
```diff
@@ -42,52 +41,45 @@ function basefiles() {
...
 # This function runs 'go fmt' and 'go vet' on eery file
 # that ends in '.go'
 function golang() {
   echo "Running go fmt and go vet"
-  find . -name "*.go" -not -path "*/istio-1.0.1/*" \
-    -not -path "*/gke-istio-shared/*" -exec go fmt {} \;
-  find . -name "*.go" -not -path "*/istio-1.0.1/*" \
-    -not -path "*/gke-istio-shared/*" -exec go vet {} \;
+  find . -name "*.go" -exec go fmt {} \;
+  find . -name "*.go" -exec go vet {} \;
```
Comment (geojaz) [inline test/make.sh]: see above

File: test/make.sh [hunk 5]
```diff
@@ -42,52 +41,45 @@ function basefiles() {
...
 }
 
 # This function runs the flake8 linter on every file
 # ending in '.py'
 function check_python() {
   echo "Running flake8"
-  find . -name "*.py" -not -path "*/istio-1.0.1/*" \
-    -not -path "*/gke-istio-shared/*" -exec flake8 {} \;
+  find . -name "*.py" -exec flake8 {} \;
```
Comment (geojaz) [inline test/make.sh]: see above

File: test/make.sh [hunk 6]
```diff
@@ -42,52 +41,45 @@ function basefiles() {
...
 }
 
 # This function runs the shellcheck linter on every
 # file ending in '.sh'
 function check_shell() {
   echo "Running shellcheck"
-  find . -name "*.sh" -not -path "*/istio-1.0.1/*" \
-    -not -path "*/gke-istio-shared/*" -exec shellcheck -x {} \;
+  find . -name "*.sh" -exec shellcheck -x {} \;
```
Comment (geojaz) [inline test/make.sh]: see above

File: test/make.sh [hunk 7]
```diff
@@ -42,52 +41,45 @@ function basefiles() {
...
 }
 
 # This function makes sure that there is no trailing whitespace
 # in any files in the project.
 # There are some exclusions
 function check_trailing_whitespace() {
   echo "The following lines have trailing whitespace"
-  grep -r '[[:blank:]]$' --exclude-dir=".terraform" --exclude-dir="istio-*" \
-    --exclude="*.png" --exclude-dir=".git" --exclude="*.pyc" .
+  grep -r '[[:blank:]]$' --exclude-dir=".terraform" --exclude="*.png" --exclude-dir=".git" --exclude="*.pyc" .
```
Comment (geojaz) [inline test/make.sh]: see above

File: test/make.sh (hunk 1 above)
Comment (chrislovecnm) [reply test/make.sh]: We need to update this in the main repo, not here

**Comment by chrislovecnm** [comment]
@geojaz we need to make changes in the main repo, not here.

**Pull request merged**